src/core/player.js

```diff
@@ -29,16 +29,30 @@ DM.provide('',
 {
     player: function(element, options)
     {
+        element = DM.$(element);
+        if (DM.Player._INSTANCES[element.id] !== undefined)
+            throw new Error("Invalid first argument sent to DM.destroy(), this element is already a player: " + element.id);
+        if (!element || element.nodeType !== Node.ELEMENT_NODE)
```

Comment:
this `if` should be in first because he's testing `!element`

current first `if` fails if `element` is `undefined`